docs/api/fiber.md

```
@@ -153,25 +150,47 @@ This distributes the incoming connections between the spawned processes and allo
 
 #### TLS
 
-TLS serves HTTPs requests from the given address using certFile and keyFile paths to as TLS certificate and key file.
+TLS serves HTTPs requests from the given address using a `*tls.Config` object returned from a specific interface.
+
+```go title="Signature"
+type ServerTLSProvider interface {
+	  ProvideServerTLS() (*tls.Config, error)
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)